src/tla2tools.ts

```diff
@@ -148,6 +181,8 @@ async function runTool(
     toolOptions: string[],
     javaOptions: string[]
 ): Promise<ToolProcessInfo> {
+    // log the arugments:
+    //console.log(toolName + ': ' + filePath + ' ' + toolOptions.join(' ') + ' ' + javaOptions.join(' '));
```

Comment:
Should this be commented?

Reply:
I meant to leave it there commented for debugging, but can also just remove it

Reply:
No need to remove it.